flow/project.py

```diff
@@ -569,6 +572,13 @@ def copy_from(cls, *other_funcs):
             @classmethod
             def after(cls, *other_funcs):
                 "True if and only if all post conditions of other operation-function(s) are met."
+                for condition_i in other_funcs:
```

Comment:
This set of nested for loops could be written in a much shorter form, and we should only need to call `_collect_operations` once at the beginning. Also the error message should be plural since `*other_funcs` can include multiple functions. Try this code (it may need slight modification).
```python
operation_functions = [operation[1] for operation in cls._parent_class._collect_operations()]
if not all(condition in operation_functions for condition in other_funcs):
    raise ValueError("The arguments to pre.after must be operation functions.")
```